Marlin/configuration_store.cpp

```diff
@@ -58,7 +58,7 @@
  *  188  M206 XYZ  home_offset (float x3)
  *
  * Mesh bed leveling:
- *  200  M420 S    active (bool)
+ *  200  M420 S    status (0 = empty, 1 = has mesh numbers)
```

Comment:
My aim was to change that comment, but I have all my Marlin dev stuff on another computer and this have slipped. And answering your other questions exhaust me. Like I see you hunt int vs bytes, but here argue enums...which I specifically avoided.